### DISCUSSION TITLE
I want to contribute (comment here)

### DISCUSSION THREAD

**Discussion by sy-b** [Anarios/return-youtube-dislike#369] [General]
Comment here if you want to contribute in ways other than that those available on GitHub.

**Comment by SpraxDev** [comment]
Hi, I'd like to provide my own dump to this project hoping to give it a new purpose.
With a friend of mine, we've started working on a similar project when YouTube announced removing the dislike button but it never hit production.

I hope that the sheer amount that I collected provides a way to verify its integrity because it can be compared to your already existing database.

The data itself originated from the official YouTube Api v3.

- - -

### Information about my humble dump

#### Numbers (amount)
Channels: `10 140`
Videos: `4 535 569`
Thumbnails: `4 453 473`

#### What data does it contain? (*italic* is probably the most interesting)
***Video***: *video_id*, *channel_id*, *published*, *last_updated* (when did I fetch the API), *views*, *likes*, *dislikes*, title, description, category_id, thumbnail (internal-id), comments (amount), duration, privacy_status, license, made_for_kids, public_stats_viewable

***Thumbnail***: sha256, image (480x360 px, jpeg)

***Channels***: *channel_id*, title, *last_updated* (when did I fetch the API), published_at, description, country, views, subscriber_count, hidden_subscriber_count (yes/no), video_count, privacy_status, linked, made_for_kids


#### Top 10 Channel countries
| Country | Channel count |
|---------|---------------|
| _unknown_  | 6 453          |
| US      | 1 053          |
| DE      | 688           |
| JP      | 205           |
| KR      | 187           |
| GB      | 126           |
| CA      | 118           |
| CN      | 114           |
| HK      | 103           |
| IN      | 80            |


#### Top 10 Video countries (by uploaded channel)
| Country     | Video count |
|-------------|---------------|
| _unknown_ | 1 043 130       |
| US          | 602 816        |
| DE          | 543 309        |
| CN          | 461 095        |
| HK          | 434 795        |
| IN          | 298 123        |
| RU          | 155 976        |
| TH          | 112 554        |
| AL          | 95 109         |
| DZ          | 83 078         |

**Reply by sy-b** [reply]
How are you planning share this?

**Reply by SpraxDev** [reply]
> How are you planning share this?

This is primarily up to the project, if there is interest in the data.
I'd probably upload it to some cloud provider with a public share link and delete it eventually - I don't think I can keep a dump of this size online for months/years.

**Reply by sy-b** [reply]
This is something that only @Anarios can handle.

**Reply by sy-b** [reply]
Can join discord discord.gg/ryd ?
He is more active there than here.
